app/views/nonprofits/payments/_details_table.html.erb

```diff
@@ -255,7 +255,7 @@
 		</tr>
     <tr>
       <td>Anonymous?</td>
-      <td><!--= if payment_details.data.supporter.anonymous (put 'Yes') (put 'No') --></td>
+      <td><!--= if payment_details.data.anonymous_supporter (put 'Yes') (put 'No') --></td>
```

Comment:
I'm not sure why, but when I was trying to read `payment_details.data.supporter.anonymous` from here it was coming empty, even though this supporter has `false` on its anonymous attribute.